Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/ShiftDriveType.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team199.Robot2018.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class ShiftDriveType extends Command {
+
+	Timer tim;
+    public ShiftDriveType() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    	tim = new Timer();
+    	tim.reset();
+    	tim.start();
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	SmartDashboard.putBoolean("Arcade Drive", !SmartDashboard.getBoolean("Arcade Drive", false));
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return tim.get() > 0.125;
```

Comment:
This command just switches the boolean value; the person who implements JUnit test will be responsible for commenting the code.

Reply:
After another look, I'm pretty sure this code is just plain broken. Looks like it will toggle the value of "Arcade Drive" repeatedly (~every 20ms) for at least 125ms, leaving it in a psuedorandom state.

On a separate note, the person who write the code is generally responsible for writing appropriate comments.

Reply:
My bad. I see it's not broken now. Still unnecessarily confusing. Moved to issue #30.